mypy/ipc.py

```diff
@@ -13,6 +13,7 @@
 import tempfile
 from types import TracebackType
 from typing import Callable, Final
+from urllib.parse import quote, unquote
```

Comment:
As a very minor optimization, what about using `codecs.encode(<bytes_data>, 'hex')` and `codecs.decode(<encoded_data>, 'hex')` instead? We wouldn't need to depend on `urllib.parse`, which is a Python package so importing it may take some time, and it would probably be a bit faster as well.

Reply:
A similar minor optimization could be to use bytes.hex() and bytes.fromhex().

Reply:
Ended up going with `codecs.encode(<bytes_data>, 'base64')`. I think it's a better encoding for this. It has a wider alphabet, but without space. Can easily change it to hex if you think it's better.

> Base64 is a scheme for converting binary data to printable ASCII characters, namely the upper- and lower-case Roman alphabet characters (A–Z, a–z), the numerals (0–9), and the "+" and "/" symbols, with the "=" symbol as a special suffix code.